zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -515,17 +519,18 @@ mod tests {
                         blocks.height IS NULL
                         AND transactions.expiry_height <= blocks_max_height.max_height
                    ) AS expired_unmined
```

Comment:
Comment on line 516: I think this may double-count memos for notes that are in both `sapling_received_notes` and `sent_notes`. (Pre-existing bug.)

Reply:
Resolved by https://github.com/zcash/librustzcash/pull/974/files#diff-54ec5e42408ac28d55641fbe89818c7a8ea21c666b6e4b79b059776661c4bce9R488